dotenv.example

```diff
@@ -5,7 +5,7 @@ SPARC_API_USERNAME=iyqu2bm3T4rDWWpnwYfEl2i6o2Sphpuq_TKvFnIOTA0
 SPARC_API_PASSWORD=qL1oZmdMi7OU7G1f0q6aCTVR6HrmoNmpZVNc8DfHqek
 FAYE_TOKEN=3bd16424a2154769a1fb4adc081b10747bdbfdbd93682032
 CWF_FAYE_HOST=localhost:9292
-DOCUMENT_ROOT=tmp/documents
+DOCUMENTS_FOLDER=documents
```

Comment:
If the reason to move the "documents" folder to the application root is to avoid accidental clearing of the tmp cache, should "/documents" be added to .gitignore?